src/client/AM.PA.MacOs/PersonalAnalytics/App/AppDelegate.swift

```diff
@@ -154,7 +155,10 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSUserNotificationCenterDele
         statusItem.menu = menu
         defaults.register(defaults: [
             AppConstants.summaryStateKey: 0,
-            AppConstants.notificationsPersistKey: true])
+            AppConstants.notificationsPersistKey: true,
+
+            // ACTIVATE/DEACTIVATE EMOTION POP-UP USING THIS SETTING
+            AppConstants.emotionPopUpActivateKey: true])
```

Comment:
please register the tracker with the TrackerManager (in the applicationDidFinishLaunching function) instead of activating/deactivating this way.